src/views/org/create.vue

```diff
@@ -0,0 +1,131 @@
+<script setup lang="ts">
+import * as api from "@/scripts/api";
+import { useRouter } from 'vue-router';
+import { useAccountStore } from '@/scripts/store';
+import { ref } from 'vue';
+import { reactive } from 'vue';
+import { useToast } from 'primevue/usetoast';
+
+
+const path = [{ label: 'New Organization' }];
+
+const router = useRouter();
+const toast = useToast();
+
+const accountStore = useAccountStore();
+if (!accountStore.isLoggedIn) {
+    toast.add({ severity: 'error', summary: 'Error', detail: 'Please login first', life: 3000 });
+    router.push('/login');
+}
+
+
+const initialValues = reactive({
+    org_name: "",
+    contact_email: "",
+    terms: false,
+});
+
+interface OrgCreateForm<T> {
+    org_name?: T;
+    contact_email?: T;
+    terms?: T;
+}
+
+const resolver = ({ values }: { values: OrgCreateForm<string> }) => {
+    const errors: OrgCreateForm<{ message: string }[]> = {};
+
+    if (!values.org_name) {
+        errors.org_name = [{ message: 'Organization Name is required.' }];
+    }
+
+    if (!values.contact_email) {
+        errors.contact_email = [{ message: 'Your Contact Email is required.' }];
+    }
+
+    if (!values.terms) {
+        errors.terms = [{ message: "You must agree to the terms and conditions." }]
+    }
+
+    return {
+        errors
+    };
+};
+
+const name = ref('');
+const email = ref('');
+const description = ref('');
+const display_name = ref('');
+
+const inProgress = ref(false);
+const onCreateOrg = async () => {
+    inProgress.value = true;
+    const res = await api.createOrganization(
+        accountStore.auth!,
+        {
+            name: name.value,
+            description: description.value,
+            display_name: display_name.value,
+        })
+    if (!res.success) {
+        toast.add({ severity: 'error', summary: 'Error', detail: res.message, life: 3000 });
+    };
+    inProgress.value = false;
+    router.push("/org/create" + res.data!.id);
```

Comment:
Why goto create page? Consider mark as `TODO`.

```suggestion
//    router.push("/org/create" + res.data!.id);
// TODO: redirect to organization home page
```